cppcache/integration/test/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ target_compile_definitions(cpp-integration-test
 target_include_directories(cpp-integration-test
   PUBLIC
    ${CMAKE_CURRENT_BINARY_DIR}
+   ${CMAKE_CURRENT_BINARY_DIR}/..
```

Comment:
Wait, what? Why is this changing?

Reply:
config.h is placed in ${CMAKE_CURRENT_BINARY_DIR}/../framework by the cmake configure step.

Reply:
Right, but you shouldn't be using `config.h` directly outside of the framework. Is there not a framework class to access these values? Similar to how we do in .NET.